docs/source/gbd2017_models/causes/ischemic_stroke/index.rst

```diff
@@ -0,0 +1,251 @@
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - Early Neonatal
+     - [0, 7 days), age_group_id=2
+   * - YLL age group end
+     - 95 plus
+     - [95, 125 years), age_group_id=235
+   * - YLD age group start
+     - Early Neonatal
+     - [0, 7 days), age_group_id=2
+   * - YLD age group end
+     - 95 plus
+     - [95, 125 years), age_group_id=235
+
+.. todo::
+
+   Describe more assumptions and limitations of the model.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+Model Assumptions and Limitations
++++++++++++++++++++++++++++++++++
+
+Cause Model Diagram
+-------------------
+
+According to GBD 2017, stroke cases are considered acute from the day of incidence of a first-ever stroke through day 28 following the event. Post, also known as chronic, stroke includes the sequelae of an acute stroke AND all recurrent stroke events. Stroke cases are considered post beginning 28 days following the occurrence of an event. Post stroke includes the sequelae of an acute stroke AND all recurrent stroke events.
+
+.. image:: cause_model_is.svg
+
+
+Data Description
+----------------
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: State Definitions
+   :widths: 1, 5, 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - **S**\ usceptible to Ischemic Stroke
+     - Simulant that has not already had an ischemic stroke event 
+   * - A
+     - **A**\ cute Ischemic Stroke
+     - Simulant that is in duration-based period starting day of incidence of a first-ever stroke through day 28 following the event
+   * - P
+     - **P**\ ost Ischemic Stroke
+     - Simulant that is in duration-based period beginning 28 days following the occurrence of a stroke event
+
+.. todo::
+
+   Discuss with the RT/SE team how to correctly assign ids into state data and transition equations, based on case definition of IS states.
+
+.. list-table:: State Data
+   :widths: 1, 5, 5, 10
+   :header-rows: 1
+
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - -
+     - cause-specific mortality rate (csmr)
+     - :math:`\frac{\text{deaths_c495}}{\text{population}}`
+     -
+   * - P
+     - excess mortality rate (emr)
+     - emr_m10837
+     - 
+   * - A
+     - excess mortality rate (emr)
+     - emr_m9310
+     - 
+   * - S
+     - excess mortality rate (emr)
+     - 0
+     - This is not applicable
+   * - P
+     - disability weight
+     - :math:`\displaystyle{\sum_{s\in \text{391}}}^{395} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}` + disability_weight_s946 :math:`\times` prevalence_s946
+     - = average disability weight over all sequelae in the post ischemic stroke state
+   * - A
+     - disability weight
+     - :math:`\displaystyle{\sum_{s\in \text{386}}}^{380} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
```

Comment:
same logic disability of Acute IS state = sum(dw_s * prev_s) / sum(prev_s) where s = 386 to 390.

Reply:
Yes, correct! I noticed that the equation above is incorrect as it says s= 386 to 380. That was a typo, and fixed the upper limit in the sum to be 390.